### PR TITLE
updating the vignette to include new functions

### PR DESCRIPTION
**Pull request by mhines-usgs**
@jread-usgs @jiwalker-usgs review?

### CODE REVIEW

File: vignettes/hazardItems.Rmd [hunk 1]
```diff
@@ -31,13 +31,17 @@ This package **hazardItems** provides services in support of coastal change haza
...
+`refreshItemThumbnail` creates a new thumbnail PNG for the requested item ID and puts it to the user specified endpoint.
+
 `addLayer` service creates a new item from a zipped shapefile from an authenticated user.
 
 `template` service creates or replaces new child items based on an input id. 
 
-`setBaseURL` and `authenticateUser` are supporting functions for directing requests and posts to the proper development, testing or production tier, and checking proper access for posting data.
+`setBaseURL` is a supporting function for directing requests and posts to the proper development, testing or production tier 
 
+`authenticateUser` checks proper access for posting data to the currently specified tier/endpoint
```
Comment (jread-usgs) [inline vignettes/hazardItems.Rmd]: `authenticateUser` actually does the authentication. `checkAuth`, which is an internal function, is the thing that checks for valid auth.

File: vignettes/hazardItems.Rmd [hunk 2]
```diff
@@ -156,5 +160,39 @@ thumb.service(serviceEndpoint)
...
+
+##Thumbnail Refresh
+
+The `refreshItemThumbnail` accepts an item ID and will verify that the item exists, before putting a newly created thumbnail in the database.
+
+To use:
+```{r,eval=FALSE}
+setBaseURL('dev') #setBaseURL("prod") in the event of production use.
+refreshItemThumbnail('CCGftiy') #or whichever itemID you wish to regenerate and put
+```
```
Comment (jread-usgs) [inline vignettes/hazardItems.Rmd]: Good. And then should we have a higher-level workflow for the various steps required to keep things current when someone changes an item or an aggregation? It includes a number of these (refreshThumb, killCache if needed, kill downloads, re-trigger downloads etc)

Comment (mhines-usgs) [inline vignettes/hazardItems.Rmd]: Ok, that sounds sensible and useful. Sort of like we did for the real-time storm bits.

File: vignettes/hazardItems.Rmd (hunk 1 above)
Comment (mhines-usgs) [inline vignettes/hazardItems.Rmd]: chhh chh chh chhanged it.

**Pull request merged**